pkg/osquery/tables/dataflattentable/tables_test.go

```diff
@@ -0,0 +1,94 @@
+package dataflattentable
+
+import (
+	"context"
+	"path/filepath"
+	"sort"
+	"testing"
+
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/stretchr/testify/require"
+)
+
+// TestDataFlattenTable_Animals tests the basic generation
+// functionality for both plist and json parsing using the mock
+// animals data.
+func TestDataFlattenTable_Animals(t *testing.T) {
+	t.Parallel()
+
+	// Test this with both plist and json
+	testTables := map[string]Table{
+		"plist": Table{dataFunc: dataflatten.PlistFile},
+		"xml":   Table{dataFunc: dataflatten.PlistFile},
```

Comment:
is this a placeholder for if/when we do a generic 'xml' table?

Reply:
xml is a valid format for plist. It's testing thge xml plist format.